core/pom.xml

```diff
@@ -30,7 +30,7 @@
 
   <groupId>org.everit.osgi</groupId>
```

Comment:
Parent must be updated to org.everit.config.oss:6.0.6 to meet the new conventions.